newIDE/app/src/InstancesEditor/index.js

```diff
@@ -578,6 +578,12 @@ export default class InstancesEditorContainer extends Component {
     this.viewPosition.scrollTo(x, y);
   }
 
+  centerView() {
+    const x = this.props.project.getMainWindowDefaultWidth() / 2 || 0;
```

Comment:
The || 0 is unnecessary, `getMainWindowDefaultWidth()` is guaranteed to return a number